monitoring-6.x.md

```diff
@@ -0,0 +1,948 @@
+
+By default, Grafana is running in anonymous read-only mode. Anyone who logs into Gravity can view but not modify the dashboards. 
+
+### Alertmanager
+
+Is a Prometheus component that handles alerts sent by client  applications such as a Prometheus server. Alertmanager handles deduplicating, grouping and routing alerts to the correct receiver integration such as an email recipient. Alertmanager exposes the cluster-internal service `alertmanager-main.monitoring.svc.cluster.local:9093`.
+
+### Satellite
+
+[Satellite](https://github.com/gravitational/satellite) is an open-source tool prepared by Gravitational that collects health information related to the Kubernetes cluster. Satellite runs on each Gravity Cluster node and has various checks assessing the health of a Cluster. Any issues detected by Satellite are shown in the output of the gravity status command.
+
+## Metrics Overview
+
+All monitoring components are running in the “monitoring” namespace in Gravity. Let’s take a look at them:
+
+```
+$ kubectl -nmonitoring get pods
+NAME                         READY   STATUS    RESTARTS   AGE
+alertmanager-main-0                    3/3     Running   0          27m
+alertmanager-main-1                    3/3     Running   0          26m
+alertmanager-main-2                    3/3     Running   0          26m
+grafana-6b645587d-chxxg                2/2     Running   0          27m
+kube-state-metrics-69594c468-wcr4g     3/3     Running   0          27m
+nethealth-4cjwh                        1/1     Running   0          26m
+node-exporter-hz972                    2/2     Running   0          27m
+prometheus-adapter-6586cf7b4f-hmwkf    1/1     Running   0          27m
+prometheus-k8s-0                       3/3     Running   1          26m
+prometheus-k8s-1                       0/3     Pending   0          26m
+prometheus-operator-7bd7d57788-mf8xn   1/1     Running   0          27m
+watcher-7b99cc55c-8qgms                1/1     Running   0          27m
+```
+
+Most of the cluster metrics are collected by Prometheus which uses the following in-cluster services:
+
+*   [node-exporter](https://github.com/prometheus/node_exporter) (collects hardware and OS metrics)
+*   [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) (collects Kubernetes resource metrics - deployments, nodes, pods)
+
+As mentioned, node-exporter collects hardware and OS metrics. There are various collectors that are dependent on each operating system. A list of the collectors can be found [here](https://github.com/prometheus/node_exporter#collectors). By default node_exporter will expose all metrics from enabled collectors. For advanced use the node_exporter can be passed an optional list of collectors to filter metrics. In Prometheus config you can use the following syntax:
+
+```
+ params:
```

Comment:
I don't think we expose a good way of modifying Prometheus configuration so I'm not sure there's value in mentioning this.